bin/generate-readme.js

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const chalk = require('chalk')
+const fetch = require('node-fetch')
+const figures = require('figures')
+const { readFileSync, writeFileSync } = require('fs')
+
+let argv = null
+const apiUrl = 'https://api.github.com'
+
+function checkpoint (message, success = true) {
+  const prefix = success ? chalk.green(figures.tick) : chalk.red(figures.cross)
+  console.info(`${prefix} ${message}`)
+}
+
+// grab files using GitHub API.
+async function getContent (owner, repo, path) {
+  const headers = {}
+  if (argv.token) headers.authorization = `token ${argv.token}`
+  const resp = await fetch(`${apiUrl}/repos/${owner}/${repo}/contents/${path}`, {
+    headers: headers
+  })
+    .then(res => {
+      if (res.status !== 200) {
```

Comment:
If you used `gaxios` it would just work this way 😛

Reply:
wasn't sure when to reach for teeny-request or gaxios ... so reached for `node-fetch` :stuck_out_tongue: